SLHCUpgradeSimulations/Configuration/python/combinedCustoms.py

```diff
@@ -6,6 +6,8 @@
 from SLHCUpgradeSimulations.Configuration.phase2TkCustomsBE5DPixel10DLHCC import customise as customiseBE5DPixel10DLHCC
 from SLHCUpgradeSimulations.Configuration.phase2TkCustomsBE5DPixel10DLHCCCooling import customise as customiseBE5DPixel10DLHCCCooling
 from SLHCUpgradeSimulations.Configuration.phase2TkCustomsBE5DPixel10DLHCCNoDefect import customise as customiseBE5DPixel10DLHCCNoDefect
+from SLHCUpgradeSimulations.Configuration.phase2TkCustomsBE5DPixel10DCoolingDefect import customise as customiseBE5DPixel10DCoolingDefect
```

Comment:
From the logs looks like it's this line, probably just needs a `git add` for the new file.